.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 phpunit.xml
 vendor
+.idea
```

Comment:
PHPStorm/IntelliJ is a major IDE with large market share, its IDE files should really be ignored. This prevents any accidental inclusions. By anyone editing this project.

Its common place at the start of a project to include a base gitignore (Github provides templates) to prevent inclusion of project files.

There is no case where a folder with the name .idea should be included.